### ISSUE TITLE
Regression on `-k` and some special characters

### ISSUE THREAD

**Issue by sereinity** [atx/wtype#26]
Since commit 33b8942 "Convert wchar_to to xkb_keysym_t properly" because it shouldn't be done with the `-k` case.

```bash
./build/wtype -k 'Tab'
```
It types a `）` (unicode `U+FF09` *Fullwidth Right Parenthesis*) instead of a tab

I locally managed to make it work again with (it's more a hack than a proper fix as all named keys should be patched):

```diff
diff --git a/main.c b/main.c
index 810d027..54b7e17 100644
--- a/main.c
+++ b/main.c
@@ -384,6 +384,7 @@ static void print_keysym_name(wchar_t wchr, FILE *f)
                { L'\n', XKB_KEY_Return },
                { L'\t', XKB_KEY_Tab },
                { L'\e', XKB_KEY_Escape },
+               { XKB_KEY_Tab, XKB_KEY_Tab },
        };

        xkb_keysym_t keysym = xkb_utf32_to_keysym(wchr);
```

**Comment by atx** [comment]
Should be fixed in 6536edf.

**Comment by sereinity** [comment]
Thanks, fixed for ma scenario.
